include/tc/core/cuda/cuda_tc_executor.h

```diff
@@ -80,14 +95,14 @@ class CudaTcExecutor : public ::tc::TcExecutor {
       const std::vector<const void*>& inputs,
       const std::vector<void*>& outputs) const;
 
-  bool hasRTCFun() {
+  bool hasRuntimeCompiledFun() override {
     return rtcFun.get() != nullptr;
   }
 
   // It is necessary to clear the RTC manually because it can throw and we
   // can't have that in the destructor.
-  void clearRTC() {
-    if (!hasRTCFun()) {
+  void clearRuntimeCompiledFun() override {
+    if (!hasRuntimeCompiledFun()) {
```

Comment:
This looks like an independent change.

Reply:
rename also to `clearRTCFunction` following the comment above.